templates/github/.github/workflows/fips.yml.j2

```diff
@@ -0,0 +1,68 @@
+{% include 'header.j2' %}
+---
+name: Pulp FIPS Nightly CI
+on:
+  {% if fips_test -%}
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # runs at 4:00 UTC daily
+    - cron: '00 4 * * *'
+  {%- else -%}
+  workflow_dispatch:
```

Comment:
Does this mean, that you can still trigger the test by hand?
Can we rename the variable to `test_fips_nightly` to reflect that?